packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts

```diff
@@ -49,17 +49,21 @@ export class ForceOrgCreateExecutor extends SfdxCommandletExecutor<
 
 export class AliasGatherer implements ParametersGatherer<Alias> {
   public async gather(): Promise<CancelResponse | ContinueResponse<Alias>> {
+    let defaultAlias = DEFAULT_ALIAS;
+    if (vscode.workspace.rootPath) {
+      defaultAlias = path.basename(vscode.workspace.rootPath);
```

Comment:
The basename _could_ have a space in the name. I don't know (just needs verification) if you can then pass that directly to the CLI.

@ntotten - Could you try including a space in the name and see what happens?